src/site/content/en/blog/layout-instability-api/index.md

```diff
@@ -236,9 +236,31 @@ const observer = new PerformanceObserver((list) => {
   }
 })
 
-observer.observe({entryTypes: ['layoutShift']});
+observer.observe({type: 'layout-shift', buffered: true});
 ```
 
+<div class="w-aside w-aside--note">
+
+**Note:** The
```

Comment:
```suggestion
The
```

Reply:
Per our chat discussion, I believe we decided to leave this in.